core-tests/src/test/java/org/mule/el/context/MessageContextTestCase.java

```diff
@@ -139,22 +141,22 @@ public void assignPayload() throws Exception
     public void payloadAsType() throws Exception
     {
         MuleMessage mockMessage = mock(MuleMessage.class);
+        MuleMessage transformedMessage = mock(MuleMessage.class, withSettings().defaultAnswer(RETURNS_DEEP_STUBS));
```

Comment:
This should do the same and is more readable: mock(MuleMessage.class, RETURNS_DEEP_STUBS);